front/app/api/graph_data_units/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { Moment } from 'moment';
+
+export const formatMoment = (moment: Moment | null | undefined) => {
+  return moment?.format('yyyy-MM-DD');
```

Comment:
Would it be useful to have it on a more global level to replace all `date.format('YYYY-MM-DD')` with it at some point?